src/ActivityRepository.js

```diff
@@ -1,21 +1,25 @@
-class ActivityRepository {
-  constructor(id, activityData) {
+import UtilityRepository from "../src/UtilityRepository";
+
+class ActivityRepository extends UtilityRepository {
+  constructor(id, activityData, sleepData, hydrationData, userData) {
+    super(userData, sleepData, hydrationData, activityData)
     this.id = id;
     this.activityData = activityData;
-    this.user = this.getUserLogs(id);
+    this.user = this.getUserLogs(id, "activityData")
   }
 
-  getUserLogs(id) {
-    return this.activityData.filter(user => user.userID === id);
-  }
+  // getUserLogs(id) {
+    // return this.activityData.filter(user => user.userID === id);
+  //   this.getUserLogs(id, this.activityData)
+  // }
 
   getUserDate(date) {
     return this.user.find(log => log.date === date);
   }
 
-  getFilteredDate(date) {
-    return this.activityData.filter(log => log.date === date);
-  }
+  // getFilteredDate(date) {
+  //   return this.activityData.filter(log => log.date === date);
+  // }
```

Comment:
Good idea leaving these here for now. It will help us track down where the old functions were being called, so that we can update them to use the new, inherited functions.